src/helpers/createBin.ts

```diff
@@ -11,10 +12,9 @@ interface BinOptions {
 	maxUsage?: number;
 }
 export async function createBin({ code, language, lifeTime, maxUsage }: BinOptions): Promise<string> {
-	const binUrl = process.env.BIN_URL!;
-	return got
-		.post(binUrl, {
-			http2: true,
+	return request
+		.post(process.env.BIN_URL!, {
+			...request,
 			followRedirect: false,
```

Comment:
this can be in the `request`, right?

Reply:
c'est pas une propriété partagée par tous, donc je dirais que non au cas où :thinking:

Reply:
dans tous les on veut **jamais** que le bot soit redérigé

Reply:
c'est pas faux